Release/16122022/NSRetail_14122022.sql

```diff
@@ -0,0 +1,1523 @@
+ALTER TABLE ITEMPRICE
+ADD BRANCHID INT NULL
+GO 
+ALTER TABLE ITEMPRICE
+ADD PARENTITEMPRICEID INT NULL
+GO
+ALTER PROCEDURE [dbo].[USP_R_ITEMMRPLIST]          
+@ITEMCODEID NVARCHAR(20)          
+AS          
+BEGIN          
+          
+	SELECT 
+		IP.ITEMPRICEID,         
+		IP.MRP,        
+		IP.SALEPRICE,      
+		IP.GSTID,      
+		GD.GSTCODE,      
+		GD.CGST,      
+		GD.SGST,      
+		GD.IGST,      
+		GD.CESS      
+	FROM ITEMPRICE IP      
+		INNER JOIN GSTDETAIL GD ON IP.GSTID = GD.GSTID      
+	WHERE ITEMCODEID = @ITEMCODEID          
+		AND IP.DELETEDDATE IS NULL
+		AND BRANCHID IS NULL
+	ORDER BY IP.ITEMPRICEID desc
+          
+END  
+GO
+ALTER PROCEDURE USP_CU_BRANCHITEMPRICE
+@ITEMPRICEID INT = 0,
+@PARENTITEMPRICEID INT,
+@SALEPRICE DECIMAL(18,2),
+@BRANCHID INT,
+@USERID INT
+AS
+BEGIN
+	
+	IF EXISTS(SELECT 1 FROM ITEMPRICE WHERE ITEMPRICEID = @PARENTITEMPRICEID AND SALEPRICE = @SALEPRICE)
+	BEGIN
+		SELECT 'Saleprice already exists! Enter different value to create new one.'
+		RETURN;
+	END
+
+	IF NOT EXISTS(SELECT 1 FROM ITEMPRICE WHERE ITEMPRICEID = @ITEMPRICEID AND SALEPRICE = @SALEPRICE)
```

Comment:
IF EXISTS(SELECT 1 FROM ITEMPRICE WHERE ITEMPRICEID = @ITEMPRICEID AND SALEPRICE <> @SALEPRICE 
 and deletedate is null)

Reply:
Changed this line